maintainers/maintainer-list.nix

```diff
@@ -1370,6 +1370,12 @@
     github = "bloomvdomino";
     githubId = 33204710;
   };
+  blueprint-freespeech = {
+    email = "ricochet@blueprintforfreespeech.net";
+    github = "blueprint-freespeech";
```

Comment:
Your github handle is pospeselr.

Reply:
Ricochet-Refresh is maintained by Blueprint for Freespech, ricochet@blueprintforfreespeech.net goes to whomever the current developer maintaining it is.

Reply:
Orgs are not supposed to be added to the maintainer-list.nix. Please create a team for them instead. maintainer-list.nix is for single accounts.